spec/spec_helper.rb

```diff
@@ -0,0 +1,42 @@
+require 'puppetlabs_spec_helper/module_spec_helper'
```

Comment:
this file is also managed by modulesync. Any of the changes still needed? Prefered way for customazation is to throw the code into a spec/spec_helper_methods.rb and include it here. Example:
https://github.com/voxpupuli/puppet-zabbix/blob/master/spec/spec_helper.rb#L15
https://github.com/voxpupuli/puppet-zabbix/blob/master/.sync.yml#L4-L5
https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/spec/spec_helper.rb#L15-L17